src/components/TermBox.vue

```diff
@@ -9,7 +9,7 @@
 				<ul v-if="hasAliases" class="wikibase-termbox__aliases">
 					<li v-for="alias in aliases"
 						class="wikibase-termbox__alias"
-						:data-separator="'wikibase-termbox-alias-separator' | message">{{ alias.value }}</li>
+						:data-separator="ALIAS_SEPARATOR( primaryLanguage )">{{ alias.value }}</li>
```

Comment:
I don't think that having to pass the (user) language every time is a solution that scales. What ever helper we build/use to get the messages should know that and apply it automatically.

Reply:
done